<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="member1Line">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="additionButton">
          <property name="text">
           <string>+</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="subtractionButton">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="multiplicationButton">
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="divisionButton">
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="member2Line">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="answerLine">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QPushButton" name="num4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="num3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="num1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="num2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="answer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>=</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="num8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="num9">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="num0">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="num7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="comma">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>,</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="num5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="num6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="clearButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(218, 2, 2);
color:rgb(255, 255, 255)</string>
          </property>
          <property name="text">
           <string>AC</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="signButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>±</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>num0</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add0()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>16</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num7</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add7()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add5()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num8</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add8()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num6</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add6()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num9</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add9()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comma</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addComma()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>answer</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>getAnswer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>additionButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subtractionButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>subtraction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>multiplicationButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>multiplication()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>divisionButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>division()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>signButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>changeSign()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add1()</slot>
  <slot>add2()</slot>
  <slot>add3()</slot>
  <slot>add4()</slot>
  <slot>add5()</slot>
  <slot>add6()</slot>
  <slot>add7()</slot>
  <slot>add8()</slot>
  <slot>add9()</slot>
  <slot>add0()</slot>
  <slot>addComma()</slot>
  <slot>getAnswer()</slot>
  <slot>addition()</slot>
  <slot>subtraction()</slot>
  <slot>multiplication()</slot>
  <slot>division()</slot>
  <slot>clear()</slot>
  <slot>changeSign()</slot>
 </slots>
</ui>
